aws/rootcause/examples/default/main.auto.tfvars

```diff
@@ -0,0 +1,7 @@
+####### BELOW ARE REQUIRED PARAMETERS FOR TERRAFORM SCRIPT #######
+sumologic_environment       = "<YOUR SUMO DEPLOYMENT>"   # Please replace <YOUR SUMO DEPLOYMENT> (including brackets) with au, ca, de, eu, jp, us2, in, fed or us1.
+sumologic_organization_id   = "<YOUR SUMO ORG ID>"       # Please replace <YOUR SUMO ORG ID> (including brackets) with your Sumo Logic Organization ID.
+sumologic_access_id         = "<YOUR SUMO ACCESS ID>"    # Please replace <YOUR SUMO ACCESS ID> (including brackets) with your Sumo Logic Access ID.
+sumologic_access_key        = "<YOUR SUMO ACCESS KEY>"   # Please replace <YOUR SUMO ACCESS KEY> (including brackets) with your Sumo Logic Access KEY.
+sumologic_collector_id      = "<YOUR SUMO COLLECTOR ID>" # Please replace <YOUR SUMO COLLECTOR ID> (including brackets) with your Sumo Logic collector ID.
+sumo_aws_account_id         = "<SUMO'S AWS ACCOUNT ID>"  # Please replace <UMO'S AWS ACCOUNT ID> (including brackets) with Sumo Logic's aws account ID.         = 926226587429
```

Comment:
Refer above comments for both "sumologic_collector_id" and "sumo_aws_account_id"